ale/drivers/ody_drivers.py

```diff
@@ -0,0 +1,130 @@
+import spiceypy as spice
+
+import ale
+from ale.base.data_naif import NaifSpice
+from ale.base.label_isis import IsisLabel
+from ale.base.type_sensor import LineScanner
+from ale.base.base import Driver
+
+import pvl
+
+class OdyThemisIrIsisLabelNaifSpiceDriver(LineScanner, IsisLabel, NaifSpice, Driver):
+    """
+    Driver for Themis IR ISIS cube
+    """
+    @property
+    def instrument_id(self):
+        inst_id = super().instrument_id
+
+        if inst_id not in ["THEMIS_IR"]:
+            raise Exception(f"{inst_id} is not a valid THEMIS instrument name. Expecting THEMIS_IR")
```

Comment:
I'd say "is not a valid THEMIS IR instrument name". This could be confusing if the ID is THEMIS_VIS.